src/util.d

```diff
@@ -627,7 +627,7 @@ void displayFileSystemErrorMessage(string message, string callingFunction) {
     addLogEntry("  Error Message:    " ~ errorMessage);
     
     // Log the calling function
-    addLogEntry("  Calling Function: " ~ callingFunction, ["verbose"]);
+    addLogEntry("  Calling Function: " ~ callingFunction);
```

Comment:
This needs to be only displayed when using --verbose .. not all the time to reduce the error output. Please revert this change

Reply:
My idea was that all error-related information valuable for traceability, such as connection details, failed reasons, and exact functions, should be recursively dumped upon encountering an error. This way, we wouldn't need to search and match in the log or attempt to reproduce something when errors occur to obtain a detailed report. Additionally, since this function is only called upon encountering an error, its impact would be minimal when everything just works fine.
Feel free to let me know if you find this approach unnecessary, as it might be influenced by the ongoing PR that will change the behavior of error handling. I'll adjust accordingly.

Reply:
The original intention behind all of the error logging was so that, when users (usually 99% of them) are running the application, they are running in a non-verbose mode due to all the Internet help guides out there.

In the early days, a number of users wanted minimal output, minimal repetitive output as well - and when error logging was introduced, it was a good way to separate out the level of error logging output between non-verbose application error logging and what occurs during verbose error logging.

I am OK with you leaving this way it is, but please in future, check with me *why* something is a certain way.

Do to this - please contact me offline via email.